cyclops-ui/src/components/pages/TemplateStore/TemplateStore.tsx

```diff
@@ -86,6 +91,28 @@ const TemplateStore = () => {
       });
   };
 
+  const checkTemplateReference = (repo: any, path: any, templateName: any) => {
```

Comment:
Can we make these strings?
```suggestion
  const checkTemplateReference = (repo: string, path: string, version: string, templateName: string) => {
```